spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/LabeledSecretNormalizedSource.java

```diff
@@ -48,7 +48,7 @@ public NormalizedSourceType type() {
 
 	@Override
 	public String target() {
-		return "Secret";
```

Comment:
I renamed this one and config map one, because I reuse them now when generating the property source name. They were use for logging purposes only, until now; from this moment these are used when generating the property source name also.